lib/pages/v1_app.dart

```diff
@@ -503,19 +499,45 @@ class _V1AppState extends State<V1App> {
                     color: Colors.grey[400],
                   ),
                 ),
-                Container(width: _width - 320),
-                AppToolTip(
-                  'Invest in what you know best at AthleteX Markets.',
-                  IconButton(
-                    onPressed: () => launchUrl(
-                      Uri.parse(
-                        'https://athletex-markets.gitbook.io/athletex-huddle/start-here/litepaper',
-                      ),
+                IconButton(
+                  onPressed: () => launchUrl(
+                    Uri.parse(
+                      'https://www.instagram.com/athletexmarkets/?hl=en',
+                    ),
+                  ),
+                  icon: FaIcon(
+                    FontAwesomeIcons.instagram,
+                    size: 25,
+                    color: Colors.grey[400],
+                  ),
+                ),
+                IconButton(
+                  onPressed: () => launchUrl(
+                    Uri.parse(
+                      'https://www.tiktok.com/@athlete_x',
                     ),
-                    icon: FaIcon(
-                      FontAwesomeIcons.circleQuestion,
-                      size: 25,
-                      color: Colors.grey[400],
+                  ),
+                  icon: FaIcon(
+                    FontAwesomeIcons.tiktok,
+                    size: 25,
+                    color: Colors.grey[400],
+                  ),
+                ),
+                FittedBox(
+                  fit: BoxFit.scaleDown,
+                  child: AppToolTip(
+                    'Invest in what you know best at AthleteX Markets.',
+                    IconButton(
+                      onPressed: () => launchUrl(
+                        Uri.parse(
+                          'https://athletex-markets.gitbook.io/athletex-huddle/start-here/litepaper',
+                        ),
+                      ),
+                      icon: FaIcon(
+                        FontAwesomeIcons.circleQuestion,
+                        size: 25,
+                        color: Colors.grey[400],
+                      ),
```

Comment:
You placed this at the wrong file and the wrong widget. It should be in `global.dart` file and the widget you need to place these are in `bottomNavBarDesktop`. Also you forked the repo which makes it inconvenient for the devs to run the app.